specs/darwin/time_machine_destinations.table

```diff
@@ -0,0 +1,12 @@
+table_name("time_machine_destinations")
+description("Drives backed up to using Time Machine")
```

Comment:
Is it possible to have locations other than drives which are backed up to? Like network shares? Should this be `Locations backed up to...`?